### ISSUE TITLE
JSON returned by /api/nodes contains a duplicate `send_bytes` key

### ISSUE THREAD

**Issue by djcrumple** [rabbitmq/rabbitmq-management#305]
Found in 3.6.5

The cluster_links returned by /api/nodes contain two send_bytes and two send_byte_details under stats. 

```
"cluster_links": [{
		"peer_addr": "172.16.13.152",
		"peer_port": 64795,
		"sock_addr": "172.16.13.150",
		"sock_port": 25672,
		"stats": {
			"send_bytes": 5141720072,
			"send_bytes_details": {
				"rate": 2049550.2833333334
			},
			"send_bytes": 13631619783,
			"send_bytes_details": {
				"rate": 2262844.4166666665
			}
		},
		"name": "rabbit@RMQTest03"
	}, {
		"peer_addr": "172.16.13.151",
		"peer_port": 64807,
		"sock_addr": "172.16.13.150",
		"sock_port": 25672,
		"stats": {
			"send_bytes": 4107425723,
			"send_bytes_details": {
				"rate": 8040.616666666667
			},
			"send_bytes": 12850589598,
			"send_bytes_details": {
				"rate": 1090.1
			}
		},
		"name": "rabbit@RMQTest02"
	}],
```

**Comment by michaelklishin** [comment]
This is https://github.com/rabbitmq/rabbitmq-management/issues/278. Please upgrade to `3.6.6`.

**Comment by michaelklishin** [comment]
Or it could be a variation thereof but if we don't have a way to reproduce this against `3.6.6`, we can't be sure. Some JSON parsers are more forgiving, so we also need to know what specific library can be used to reproduce.

**Comment by michaelklishin** [comment]
With #236 about to be merged this may or may not be relevant for `3.6.7` already. @dcorbacho and @kjnilsson can tell :)

**Comment by michaelklishin** [comment]
I also suspect that the field was supposed to be named `sent_bytes` and not `send_bytes`.

**Comment by dcorbacho** [comment]
Fixed in https://github.com/rabbitmq/rabbitmq-management/blob/rabbitmq-management-236/src/rabbit_mgmt_stats.erl#L332

@michaelklishin That field is named `send_bytes` since at least 3.5.7

**Comment by michaelklishin** [comment]
@dcorbacho yes, seems to be introduced in https://github.com/rabbitmq/rabbitmq-management/commit/dc4a42a7cb5f54fb690c8221d7a8209b366c5f69 :( Alright, let's leave it as is then.

**Comment by michaelklishin** [comment]
FTR, this was fixed as part of #236.
